extensions/slack-status/src/actions.tsx

```diff
@@ -70,7 +72,7 @@ export function SetStatusWithAIAction(props: { statusText: string; mutate: Mutat
               `You help a Slack user set their status.
               
               **Respond with a JSON object with the following attributes:**
-              - "text": a string value for status text, should be short and sweet, with no punctuation, e.g. "Working out", "Listening to Drake's new album", "Coffe break". It should not include the status duration for example "Working out" instead of "Working out for 2 hours" or "Working out until tomorrow".
+              - "text": a string value for status text, should be short and sweet, with no punctuation, e.g. "Working out", "Listening to Drake's new album", "Coffee break". It should not include the status duration for example "Working out" instead of "Working out for 2 hours" or "Working out until tomorrow".
```

Comment:
Another snuck-in typo fix.

I considered updating the AI action to be able to set DND, but it seemed unlikely to be used and would add complexity, so I skipped it.